rebar.config

```diff
@@ -57,10 +57,10 @@
 {deps,
  [
   {lager, ".*", {git, "git://github.com/basho/lager", {branch, "1.2"}}},
-  {jsx, "1\.4\..*", {git, "git://github.com/talentdeficit/jsx", "v1.4.2"}},
+  {jsx, "1\.4\..*", {git, "git://github.com/talentdeficit/jsx", {tag, "v1.4.2"}}},
 
   %% for testing
-  {proper, "1\.1", {git, "git://github.com/manopapad/proper", "1b773ee"}}, %"v1.1"}},
-  {meck, "0.8.2", {git, "git://github.com/eproxus/meck.git", "dde7590"}}
+  {proper, "1\.1", {git, "git://github.com/manopapad/proper", {ref, "1b773ee"}}},
```

Comment:
I didn't take it as an offense. I was just informing you that it could take
a while until I update the PR.

Philip Müller <notifications@github.com> schrieb am Fr., 19. Feb. 2016
17:33:

> In rebar.config
> <https://github.com/disco-framework/disco/pull/16#discussion_r53483135>:
>
> >
> >    %% for testing
> > -  {proper, "1\.1", {git, "git://github.com/manopapad/proper", "1b773ee"}}, %"v1.1"}},
> > -  {meck, "0.8.2", {git, "git://github.com/eproxus/meck.git", "dde7590"}}
> > +  {proper, "1\.1", {git, "git://github.com/manopapad/proper", {ref, "1b773ee"}}},
>
> Hey, no offense intended! Thanks for your work on this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/disco-framework/disco/pull/16/files#r53483135>.
>